docker-cloud/migration/kube-primer.md

```diff
@@ -94,7 +94,7 @@ This command deploys a Docker application, named `test-app`, from a YAML configu
 $ docker stack deploy -c app1.yml test-app
 ```
 
-This command deploys a Kubernetes application from a YAML manifest file called `k8s-app1.yml`:
+This command deploys a Kubernetes application from a YAML manifest file called `k8s-app.yml`:
```

Comment:
This change is not correct. Line 94 tells users to create a file names `app1.yml`, so lines 94, 97, 100 should use `app1.yml`.

Reply:
Please see new PR below for the suggested change. :+1: